src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Todo (props) {
+    let completed = props.eachTask.completed ? "completed" : "";
+    return (
+        <h3 className={`task-style ${completed}`} onClick={() => props.clickHandler(props.index)}>{props.eachTask.tasks}</h3>
```

Comment:
It slipped my mind at the time, but since each todo item has an `id`, that would be a safer approach as opposed to using `index`.
```suggestion
        <h3 className={`task-style ${completed}`} onClick={() => props.clickHandler(props.id)}>{props.eachTask.tasks}</h3>
```